Sources/SwiftKuery/Table.swift

```diff
@@ -16,9 +16,32 @@
 
 // MARK: Table
 
-/// Subclasses of the Table class are metadata describing a table in a relational database that you want to work with.
+/**
+ The `Table` class is used to represent a specific table from an SQL database in swift.
+ This class details the table name, contains the `Column` instances present inside the table and provides methods to build SQL String descriptions for the various database engines.
+ ### Usage Example: ###
+ In this example, a ToDo table class matching a table stored in an SQL database is defined.
+ The "ToDoTable" class contains the table name and three instances of the `Column` class.
```

Comment:
Please can we cut down the wording in here a little as I don't think we need to describe this in so much detail. How about...
"In this example, we define a `ToDoTable` class which maps to an existing table in a SQL database. An instance of the `ToDoTable` class is then initialized and used in a SQL SELECT query to retrieve the data from the table within the external database."